content/doc/book/managing/index.adoc

```diff
@@ -13,15 +13,146 @@ endif::[]
+that extend the functionality of Jenkins.
+
+link:nodes.adoc[Manage Nodes and Clouds]::
+Add, remove, control, and monitor the nodes used for the agents on which build jobs run.
+
+link:casc.adoc[Configuration as Code]::
+Configure your Jenkins instance using a human-readable YAML file rather than the UI.
+This is an optional feature that appears in this group
+only when the plugin is installed on your controller.
+
+== Security group
+
+Screens for configuring security features for your Jenkins instance.
+See link:/doc/book/security/[Securing Jenkins] for general information
+about managing Jenkins security.
+
+link:system-configuration.adoc[Configure Global Security]::
+Set configuration parameters that secure your Jenkins instance.
+
+link:/doc/book/using/using-credentials/#adding-new-global-credentials[Manage Credentials]::
+Configure the credentials that provide secure access
+to third-party sites and applications that interact with Jenkins.
+
+*Configure Credential Providers*::
+Configure credential providers and types
+
+link:users.adoc[Manage Users]::
+Manage users defined in the Jenkins user database.
+This is not used if you use a different security realm such as LDAP or AD.
+
+== Status Information group
+
+link:system-info.adoc[System Information]::
+Displays information about the Jenkins environment.
+
+link:/doc/book/system-administration/viewing-logs/[System Log]::
+Jenkins log that contains all `java.uil.logging` output related to Jenkins.
+
+*Load Statistics*::
+Displays information about resource utilization on you Jenkins instance.
+
+*Disk usage*::
+Simple disk usage estimation
+
+link:about-jenkins.adoc[About Jenkins]::
+Provides version and license information for your Jenkins instance.
+
+== Troubleshooting group
+
+*Manage Old Data*::
+Remove configuraion information related to plugins that have been removed from the instance.
+
+== Tools and Actions group
+
+Screens for common management tasks
+and management tools that enable you to do administrative tasks without using the UI.
+
+*Reload Configuration from Disk*::
+Discard all data that is loaded in memory and reload everything from the file system.
+This is useful when you modify configuration files directly on disk.
+
+link:cli-adoc[Jenkins CLI]::
+How to use the Jenkins CLI from a shell or script.
+
+link:/doc/book/managing/script-console/[Script Console]::
+Execute an Apache Groovy script for administration, troubleshooting, and diagnostics.
+
+*Prepare for Shutdown*::
+Prevents new builds from starting so that the system can be shut down safely.
+
+== Uncategorized group
+
+Screens for monitoring the Jenkins controller and agents
+and for launching build agents as Docker containers.
+
+*Monitoring of Jenkins controller*::
+Monitors memory, cpu, http requests and more on the Jenkins controller.
+
+*Monitoring of Jenkins agents*::
+Monitors builds, the build queue, and Jenkins agents.
+
+*Docker*::
+Plugin for launching build agents as Docker containers.
```

Comment:
Uncategorized is used for those plugins that have not yet declared the category of their page.

```suggestion
Screens for plugins that have not yet declared the category of their page.
```